inc/templates.php

```diff
@@ -401,7 +401,7 @@ function hydrate_components( array $components ) {
 		if ( 'irving/text' === $component->get_name() ) {
 			$hydrated[] = $component->get_config( 'content' );
 		} else {
-			$hydrated[] = $component->jsonSerialize();
+			$hydrated[] = $component;
```

Comment:
I don't understand this change. Is serialization happening elsewhere now?

Reply:
Serialization will happen automatically when the endpoint is spit out, but converting the components back to arrays before the output is ready is causing issues where I'm trying to modify something, and in some cases it's an array, and in others it's an object. Additionally, when attempting to run the hydration process on something that's already been hydrated, I'm running into issues.

I understand what you're trying to accomplish here, but it's not working well during my experiments, and making this change has cleared things up.